scripts/test/compareBenchmarkScriptTest.py

```diff
@@ -1,31 +1,50 @@
 #!/usr/bin/env python3
 
 import json
+import math
+import re
 import subprocess
 
 # This helper test class takes two JSON result files, runs the compare_benchmarks.py script, and checks that the output
 # is as expected. It is called from benchmark tests such as hyriseBenchmarkTPCC_test.py.
 
 
-# The next two functions check for string matches instead of using math.isclose() since extracting values from
-# ANSI-colored text turned out to be too cumbersome.
+# Function removes any ansi escape sequences, which we use for color coding.
+# Gratefully taken from https://stackoverflow.com/a/14693789/1147726
+def clean_ansi_escape_sequences(input_string):
+    ansi_escape = re.compile(r'''
+    \x1B  # ESC
+    (?:   # 7-bit C1 Fe (except CSI)
+        [@-Z\\-_]
+    |     # or [ for CSI, followed by a control sequence
+        \[
+        [0-?]*  # Parameter bytes
+        [ -/]*  # Intermediate bytes
+        [@-~]   # Final byte
+    )
+    ''', re.VERBOSE)
+    return ansi_escape.sub('', input_string)
+
+
+# As different precisions are used (latency for successful and unsucceful runs, throughput), string comparisons with
```

Comment:
Latency and throughput are no precisions.

Reply:
`As different precisions are used (i.e., precision for rounded latencies is different from rounding precision of throughputs)`

Better?